controllers/drcluster_controller_test.go

```diff
@@ -186,6 +186,42 @@ func inspectClusterManifestSubscriptionCSV(match bool, value string, drcluster *
 	}
 }
 
+func updateDRClusterParameters(clusterName string, clusterFence ramen.ClusterFenceState,
+	cidrs []string, s3ProfileName string,
+) *ramen.DRCluster {
```

Comment:
This returns a pointer to ramen.DRCluster...

Reply:
yes, the updated drCluster. The latest one